src/compounds/LinkedButton/LinkedButton.jsx

```diff
@@ -29,7 +31,10 @@ const { onClick, ...remainingPropTypes } = Button.propTypes
 LinkedButton.propTypes = {
   buttonProps: PropTypes.shape(remainingPropTypes).isRequired,
   addClass: PropTypes.string,
-  path: PropTypes.string.isRequired,
+  path: PropTypes.shape({
```

Comment:
More just a question - Would `PropTypes.exact` be helpful here?

Reply:
good question. if I recall correctly, when I first started the library I decided on `shape` over `exact` because I was setting it on an image and I wanted to let us pass other props other than the required ones I'd listed. 

however...it's become a kind of habit to use `shape` all of the time, where in some instances `exact` may be better used. I'll reevaluate them all later.

thanks! 😁